scripts/logger-util.js

```diff
@@ -0,0 +1,24 @@
+const RESET = "\x1b[0m";
+const BOLD = "\x1b[1m";
+const CYAN_BOLD = `${BOLD}\x1b[36m%s${RESET}`;
+const RED = `\x1b[31m%s${RESET}`;
+const YELLOW = `\x1b[33m%s${RESET}`;
+
+module.exports = {
```

Comment:
Decided to move the color modifications into their own util. If we find that we want to change these colors going forward, this will be an easy way to do it.